gs.h

```diff
@@ -483,7 +483,7 @@
 /*========================
 // Defines
 ========================*/
-
+;
```

Comment:
Remove semi-colon here

Reply:
There is a a weird issue where tcc does not parse the initial headers properly if the semi colon is not there. The alternative is hiding it a few lines above or having tcc users add the semi-colon there.